src/core/binding.ts

```diff
@@ -82,6 +82,10 @@ export class Binding {
   private willBeInvokedByEvent(event: Event): boolean {
     const eventTarget = event.target
 
+    if (this.action.shouldIgnoreGlobalEvent(event, this.element)) {
```

Comment:
```suggestion
    if (this.action.shouldIgnoreGlobalEvent(event, this.action.element)) {
```

I believe this is all that will be needed to change the behaviour (see parent PR comment) so that the contains element is based on where the action was set up.

Also, I am not sure if I should be using something like the code below in this file.

```
this.scope.containsElement(this.action.element)
```

I am not 100% sure on the implications of using/not using the `this.scope`.